src/renderer/common/components/dialog/publicationInfos/formatPublisherDate.tsx

```diff
@@ -26,9 +26,9 @@ export const FormatPublisherDate: React.FC<IProps> = (props) => {
         const date = moment(publication.publishedAt).format("L");
         if (date) {
             formatedPublishedDateComponent = (
-                <p>
+                <div>
```

Comment:
I'm not an expert, but I've heard that a div alone is misunderstood by screen reader.
@danielweck what do you think ?

Reply:
It's just for my curiosity :) .. not a strict review comment

Reply:
No, it is fine to use `div` as non-semantic, purely structural dividers. :)